snapcraft/internal/sources/errors.py

```diff
@@ -87,6 +87,15 @@ class InvalidDebError(SnapcraftSourceError):
     )
 
 
+class InvalidSnapError(SnapcraftSourceError):
+
+    fmt = (
+        "The snap file used does not contain valid data. "
+        "Ensure a proper snap file is passed for .snap files "
```

Comment:
The wording here sounds a bit strange. Perhaps something like "The snap file does not contain valid data. Ensure the source lists a proper snap file" could be better?